Source/WebCore/dom/Node.h

```diff
@@ -193,7 +193,7 @@ class Node : public EventTarget, public CanMakeCheckedPtr<Node> {
     virtual const AtomString& namespaceURI() const;
     virtual const AtomString& prefix() const;
     virtual ExceptionOr<void> setPrefix(const AtomString&);
-    WEBCORE_EXPORT void normalize();
+    WEBCORE_EXPORT ExceptionOr<void> normalize();
```

Comment:
Why do we have `WEBCORE_EXPORT` on this? Are there any callers outside WebCore that need to be updated?

Reply:
GTK+ calls it in `webkit_dom_node_normalize`.